scrapers/Brazzers.yml

```diff
@@ -1,23 +1,32 @@
-name: "Brazzers"
+name: 'brazzers'
 sceneByURL:
   - action: scrapeXPath
     url:
-      - https://www.brazzers.com/video/
+      - brazzers.com/video
+      - brazzers.com/scenes
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     scene:
       Title: //h2[@class="font-secondary sc-1b6bgon-1 kOFSRa"]/text()
+      Studio:
+        Name: 
+            selector: //a[@class="vdkjux-5 gIliaY"]/text()
```

Comment:
the `selector` should have a couple of spaces less
`s` from **selector** should be under the `m` from **Name**

also the 
```yaml
# Remove '#' if you want get the studio without space
```
should go here ( above the postProcess ) comment